### ISSUE TITLE
Mining drones will break blocks outside of their designated space

### ISSUE THREAD

**Issue by Tobanyi** [mcenderdragon/Futurepack-API#886]
**Describe the bug**
Mining drone is breaking blocks outside of its zone.

**To Reproduce**
1. Set up a mining drone with a zone station
2. Activate the drone
3. The drone will sometimes break blocks outside of the mining zone.

**Screenshots**
https://imgur.com/a/jt30LqB

**Additional context**
This seems to only happen with blocks with a short mining duration.  It looks like the drone is trying to mine any block in front of it when it is moving, not just what is in the zone.

**Version**
Minecraft Version: 1.18.2
Forge Version: 40.1.25
Futurepack Version: Futurepack-1.18.2-33.0.7268

**Logs**
N/A

**Comment by mcenderdragon** [comment]
yeah this one is very old and also ahappen in older version if the blocks can be broken easily.

**Comment by Tobanyi** [comment]
Ah, I guess I never noticed it before.

**Comment by Yirok42** [comment]
It happens even more if you put 4 drones on the same zone. A thing I do sometimes but i have not timed to see if it is any faster than using one.
